Gemfile

```diff
@@ -7,5 +7,5 @@
 source 'https://rubygems.org'
 
 gem 'rake', '~> 13.0'
-gem 'rubocop', '~> 1.62'
+gem 'rubocop', '~> 1.64.0'
```

Comment:
Do you intentionally only want to allow installing bug releases going forward?

Reply:
Thanks for noticing this. I'm not sure. 

In the previous PR, I changed it from allowing only bug releases to allowing minor version updates. That caused our main build to fail. If we only allow bug releases, we are less likely to have that problem, and may have dependabot open PRs first. 

I don't feel strongly either way, they both have their pros/cons. What do you think?